test/fixtures/users.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```

Comment:
If we are going to build fixtures I'd prefer to have them be valid and have context. I find the easiest thing is to find a theme and just go with it. 

E.g. I like to do Harrison Ford characters for my users:
```
han_solo:
  han.solo@rebellion.com
  ISh0TF!rst55

indiana_jones
  indiana.jones@artifacts.com
  IH@teSn@kes66
```

Really doesn't matter the theme but it does help keep track of what is in your fixtures when using them. As they grow you will find having more realistic data is far easier to work with.